### ISSUE TITLE
请问yolov6如何修改代码以在训练结束前若干epoch关闭数据增强

### ISSUE THREAD

**Issue by JiaPai12138** [meituan/YOLOv6#415]
rt

**Comment by iodncookie** [comment]
i try in engine.py file modify prepare_for_steps functions，eg：
`def prepare_for_steps(self):
        if self.epoch > self.start_epoch:
            self.scheduler.step()
        # if self.epoch > 380:
        #     self.cfg.data_aug['mosaic'] = 0.0
        #     self.train_loader, self.val_loader = self.get_data_loader(self.args, self.cfg, self.data_dict)`

**Comment by mtjhl** [comment]
> i try in engine.py file modify prepare_for_steps functions，eg： `def prepare_for_steps(self):
> 
> ```
>     if self.epoch > self.start_epoch:
>         self.scheduler.step()
>     # if self.epoch > 380:
>     #     self.cfg.data_aug['mosaic'] = 0.0
>     #     self.train_loader, self.val_loader = self.get_data_loader(self.args, self.cfg, self.data_dict)
> ```
This  method is fine. @JiaPai12138

**Comment by JiaPai12138** [comment]
Thanks!
